### PR TITLE
Update observability best practices

### PR DESCRIPTION
**Pull request by jacob-delgado**
Istio no longer ships with Prometheus. Reference the Prometheus addon.

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/best-practices/observability/index.md [hunk 1]
```diff
@@ -13,14 +13,10 @@ The recommended approach for production-scale monitoring of Istio meshes with Pr
...
-Prometheus is intentionally deployed with a very short retention window (6 hours). The
-default Prometheus deployment is also configured to collect metrics from each Envoy proxy
-running in the mesh, augmenting each metric with a set of labels about their origin (`instance`,
-`pod`, and `namespace`).
-
-While the default configuration is well-suited for small clusters and monitoring for short time horizons,
+Istio no longer ships with Prometheus in its default profile. To enable Prometheus please see
+the [Prometheus integration](../../../ops/integrations/prometheus/) addon documentation.
+
+While the `Quick Start` installation is well-suited for small clusters and monitoring for short time horizons,
```
Comment (ericvn) [inline content/en/docs/ops/best-practices/observability/index.md:19]: Not sure what `Quick Start` is. Maybe you are referring the the `Getting Started` installation. Or we could say `default configuration profile`.

Comment (frankbu) [reply content/en/docs/ops/best-practices/observability/index.md:19]: @douglas-reid PTAL

Comment (jacob-delgado) [reply content/en/docs/ops/best-practices/observability/index.md:19]: I was referencing https://istio.io/latest/docs/ops/integrations/prometheus/#option-1-quick-start which is mentioned in the link to the Prometheus integration. The phrasing could be better.

Comment (douglas-reid) [reply content/en/docs/ops/best-practices/observability/index.md:19]: Should we call it `sample` install ?

Comment (jacob-delgado) [reply content/en/docs/ops/best-practices/observability/index.md]: We no longer install Prometheus in the default nor in the demo profile. For the users to test prometheus they have to follow the addons integration page. It's not difficult, but no longer an option for testing out the functionality easily.

Comment (ericvn) [reply content/en/docs/ops/best-practices/observability/index.md]: I'm wondering if this paragraph would be a better tip/info on the prometheus add-on page. We do have a very small blurb there.

File: content/en/docs/ops/best-practices/observability/index.md [hunk 2]
```diff
@@ -13,17 +13,14 @@ The recommended approach for production-scale monitoring of Istio meshes with Pr
...
-the introduced labels can increase metrics cardinality, requiring a large amount of storage. And, when trying
-to identify trends and differences in traffic over time, access to historical data can be paramount.
+Istio no longer ships with Prometheus in its default profile. To enable Prometheus please see
+the [Prometheus integration](../../../ops/integrations/prometheus/) addon documentation.
+
+While the `Option 1: Quick Start` installation mentioned in the [Prometheus integration](../../../ops/integrations/prometheus/)
+guide is well-suited for small clusters and monitoring for short time horizons, it is not suitable for large-scale
+meshes or monitoring over a period of days or weeks. In particular,  the introduced labels can increase metrics
+cardinality, requiring a large amount of storage. And, when trying to identify trends and differences in traffic over
+time, access to historical data can be paramount.
```
Comment (frankbu) [inline content/en/docs/ops/best-practices/observability/index.md:23]:
```suggestion
Although installing Istio does not deploy [Prometheus](http://prometheus.io) by default, the
[Getting Started](/docs/setup/getting-started/) instructions install the `Option 1: Quick Start` deployment
of Prometheus described in the [Prometheus integration guide](../../../ops/integrations/prometheus/).
This deployment of Prometheus is intentionally configured with a very short retention window (6 hours). The
quick-start Prometheus deployment is also configured to collect metrics from each Envoy proxy
running in the mesh, augmenting each metric with a set of labels about their origin (`instance`,
`pod`, and `namespace`).
 
While the quick-start configuration is well-suited for small clusters and monitoring for short time horizons,
it is not suitable for large-scale meshes or monitoring over a period of days or weeks. In particular,
the introduced labels can increase metrics cardinality, requiring a large amount of storage. And, when trying
to identify trends and differences in traffic over time, access to historical data can be paramount.
```

File: content/en/docs/ops/best-practices/observability/index.md (hunk 1 above)
Comment (jacob-delgado) [reply content/en/docs/ops/best-practices/observability/index.md]: Moved to the addons page for Prometheus. Combined Frank's suggestion too.

**Review by ericvn — APPROVED** [review]

**Pull request merged**